packages/api/src/api/videos.ts

```diff
@@ -0,0 +1,34 @@
+import {Router} from 'express';
+import { Video } from '../entities/Video';
+import logger from '../logger';
+
+export const videos = Router();
+
+// Get all videos
+videos.get('', async (req, res) => {
+    try {
+        const videosRepository = req.entityManager.getRepository(Video);
+        const videosList = await videosRepository.findAll();
+
+        res.statusCode = 200;
+        res.send(videosList);
+    } catch (error) {
+        logger.error('There was an issue geting all videos: ', error);
+        res.statusCode = 500;
+        res.send('There was an issue geting all videos');
+    };
+});
+
+// Get video by id
+videos.get('/:video_id', async (req, res) => {
+    const { video_id } = req.params;
```

Comment:
What happens if a `video_id` isn't provided? 🤔

Reply:
If video_id isn't provided the path resolves to the path "api/videos/" which returns all videos.  I believe this is the desired outcome in this situation.

Reply:
Oh I'm dumb 🤦‍♂️ thought this was a query param, not a path param... carry on 🤣